Sources/Proxy/Core/AbstractRepresentationProxy/index.js

```diff
@@ -302,7 +302,7 @@ function extend(publicAPI, model, initialValues = {}) {
   Object.assign(model, DEFAULT_VALUES, initialValues);
 
   macro.obj(publicAPI, model);
-  macro.get(publicAPI, model, ['input', 'actors', 'volumes']);
+  macro.get(publicAPI, model, ['input', 'mapper', 'actors', 'volumes']);
```

Comment:
But if there is no mapper?

Reply:
Meaning `model.mapper` is `undefined` as it is not used or set?

Reply:
Can a representation not have a mapper?? and if it is undefined won't that just return undefined?

Reply:
yes a rep can be anything. But I guess that's ok to return undefined. 
Just wanted to mention that you should not expect to always have a mapper in a rep.

Reply:
don't representations always have a mapper? And even if it didn't, wouldn't it just return `undefined` and then whoever called this need to handle it?

Reply:
your call, i don't need it anymore for my use case (and could access it by reference instead than get)